kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/cloud/common/utils/AsyncIOAssertions.scala

```diff
@@ -0,0 +1,17 @@
+package io.lenses.streamreactor.connect.cloud.common.utils
+
+import cats.effect.IO
+
+/**
+  * This is a workaround for being unable to use AsyncIOSpec in some tests due to their superclasses using some other non-IO spec.
```

Comment:
should we separate the IO tests?

Reply:
I am not sure the best approach.  Ideally we could revisit `CloudPlatformEmulatorSuite` and upgrade it to the IO world, but it has a lot of subclasses that we'd also have to touch.  So I avoided it and did this 😁